test/Sentry.Tests/Internals/GarbageCollectionMonitorTests.cs

```diff
@@ -2,9 +2,11 @@ namespace Sentry.Tests.Internals;
 
 public class GarbageCollectionMonitorTests
 {
-    [Fact]
+    [SkippableFact]
     public async Task MonitorGarbageCollection_TaskCancelled_CancelsFullGCNotification()
     {
+        Skip.If(!RuntimeInformation.IsOSPlatform(OSPlatform.Windows), "These tests may be hanging in CI on Windows");
```

Comment:
Isn't the condition reversed? (Also, what TFM is targeted in tests? Can this use `OperatingSystem.IsWindows()`?)

Reply:
Hah - good spotting. I just put that in last night before going to sleep...

Reply:
If CI was passing with that, then I guess tests are not that flakey after all 😅 

Thanks @filipnavara as always